cypress/support/fragments/circulation-log/searchPane.js

```diff
@@ -5,10 +5,10 @@ import DateTools from '../../utils/dateTools';
 
 export default {
   searchByItemBarcode(barcode) {
-    return cy.do([
+    cy.do([
       TextField({ name: 'itemBarcode' }).fillIn(barcode),
-      Accordion({ id: 'accordion_5' }).find(Button('Apply')).click()
     ]);
+    cy.get('[class^="button-"][type="submit"]').first().click();
```

Comment:
let's switch to interactors. You can add new filters to implement required logic

Reply:
How can I pick first button from 2 equals button with interactor?
These button are equal with class, attributes and text
![image](https://user-images.githubusercontent.com/26041116/156886179-9d4631ee-c697-4f3a-a8ac-aa7e76c35fcd.png)

Reply:
you can try to work through sections
![image](https://user-images.githubusercontent.com/93538670/156886346-8af2a219-6caf-40d0-ae4f-e84033ab2c1d.png)

Reply:
Can we sure that accordion_5 is constant with other environment? I replaced it because sometimes my test was failed, and this section had id like accordion_136, accordion_104 etc

Reply:
you can check this by yourself(https://wiki.folio.org/display/DQA/Manage+id%27s+of+web+elements) or by support of frontend developers.